punishments.js

```diff
@@ -1050,4 +1050,4 @@ Punishments.monitorRoomPunishments = function (user) {
 
 		Monitor.log(`[PunishmentMonitor] ${user.name} currently has punishments in ${punishments.length} rooms: ${punishmentText}`);
 	}
-};
\ No newline at end of file
+};
```

Comment:
@bumbadadabum, this actually _is_ a problem though; be more careful about trailing whitespace...